### PR TITLE
e2e

### PR DESCRIPTION
**Pull request by kennyworkman**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,7 +37,9 @@ RUN curl -s https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.
 
 COPY gentrome.sh /root/gentrome.sh
 
-RUN python3 -m pip install --upgrade multiqc matplotlib numpy scipy lgenome latch
+RUN python3 -m pip install --upgrade multiqc matplotlib numpy scipy lgenome
+
+RUN python3 -m pip install latch==1.13.3
```
Comment (ayushkamat) [inline Dockerfile]: why pin the version to 1.13.3

File: wf/__init__.py [hunk 2]
```diff
@@ -10,18 +10,16 @@
...
 from enum import Enum
 from pathlib import Path
-from typing import Iterable, List, Optional, Tuple, Union
+from typing import Annotated, Iterable, List, Optional, Tuple, Union
 
 import lgenome
 from dataclasses_json import dataclass_json
 from flytekit import task
+from flytekit.core.annotation import FlyteAnnotation
+from flytekit.core.launch_plan import reference_launch_plan
```
Comment (ayushkamat) [inline wf/__init__.py]: switch this to workflow_reference from `latch`

File: wf/__init__.py [hunk 3]
```diff
@@ -712,14 +777,18 @@ def rnaseq(
...
+    ] = [],
     custom_gtf: Optional[LatchFile] = None,
     custom_ref_genome: Optional[LatchFile] = None,
     custom_ref_trans: Optional[LatchFile] = None,
     star_index: Optional[LatchFile] = None,
     salmon_index: Optional[LatchFile] = None,
     save_indices: bool = False,
     custom_output_dir: Optional[LatchDir] = None,
-) -> List[LatchFile]:
+):
```
Comment (ayushkamat) [inline wf/__init__.py:752]: no returns?

**Review by ayushkamat — APPROVED** [review]
just a few questions (see comments), but ow looks good

File: wf/__init__.py (hunk 3 above)
Comment (kennyworkman) [reply wf/__init__.py:752]: yes, we no longer rely on returns from workflow, only tasks

File: Dockerfile (hunk 1 above)
Comment (kennyworkman) [reply Dockerfile]: old thing from debug

**Pull request merged**